client/battle/CBattleInterface.cpp

```diff
@@ -3206,18 +3213,58 @@ void CBattleInterface::showProjectiles(SDL_Surface *to)
 				continue; // wait...
 		}
 
-		size_t group = it->reverse ? 1 : 0;
-		auto image = idToProjectile[it->creID]->getImage(it->frameNum, group, true);
+		if ( idToProjectile.count(it->creID) != 0)

```

Comment:
We do not use spaces after if, for, while according to style guidelines

```suggestion
		if(idToProjectile.count(it->creID) != 0)

```